commons/src/main/java/gov/cms/qpp/conversion/util/MeasuredInputStreamSupplier.java

```diff
@@ -0,0 +1,43 @@
+package gov.cms.qpp.conversion.util;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.util.Objects;
+import java.util.function.Supplier;
+
+import org.apache.commons.io.IOUtils;
+
```

Comment:
javadocs

Reply:
Dunno why your review wasn't autoremoved when I addressed this, but javadocs are now present.